src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1470,3 +1470,11 @@ def list_consumption_locations():
     client = web_client_factory()
     regions = client.list_geo_regions(sku='Dynamic')
     return [{'name': x.name.lower().replace(" ", "")} for x in regions]
+
+def list_locations(sku):
+    client = web_client_factory()
+    sku_to_check = _get_sku_name(sku)
+    if sku.lower().endswith('v2'):
+        sku_to_check = "{}V2".format(sku_to_check)
+    regions = client.list_geo_regions(sku_to_check)
+    return [{'name': x.name.lower().replace(" ", "")} for x in regions]
```

Comment:
Is this a breaking change in the sense that it changes the output data's format?

Reply:
The output format will be the same.